primitives/src/parachain.rs

```diff
@@ -706,6 +752,15 @@ pub enum CompactStatement {
 /// A signed compact statement, suitable to be sent to the chain.
 pub type SignedStatement = Signed<CompactStatement>;
 
+/// Result of the validation of the candidate.
+#[derive(RuntimeDebug)]
+pub enum ValidationResult {
```

Comment:
This can probably go in `node-primitives` as it does not need to enter the runtime.